Include/object.h

```diff
@@ -142,8 +142,8 @@ static inline Py_region_ptr_with_tags_t Py_region_ptr_with_tags(Py_region_ptr_t
     return (Py_region_ptr_with_tags_t) { region };
 }
 
-#define _Py_DEFAULT_REGION ((Py_region_ptr_t)0)
-#define _Py_IMMUTABLE ((Py_region_ptr_t)1)
+#define _Py_DEFAULT_REGION ((Py_region_ptr_t)1<<3)
```

Comment:
Is the 4th bit necessary? Why not the 3rd. Do we really have 16 byte alignment?

Reply:
I've reverted this back to zero. I can't remember why I believed I needed to give it a different value :shrug: